src/osx/cocoa/filedlg.mm

```diff
@@ -530,7 +601,11 @@
         }
         else
         {
-            [oPanel setAllowedFileTypes: types];
+            OpenSavePanelDelegate* delegate = [[OpenSavePanelDelegate alloc] init];
+            [delegate setWiladcard: m_wildCard];
```

Comment:
`setWiladcard` -> `setWildcard` ?

Reply:
Pffft.

Reply:
I pushed an update to the branch and it seems to have automatically added it.  I guess the PR is for the branch and not the commit?

Reply:
Correct - any commits you push to the branch get automatically added to the PR.  Usually when merging, Vadim or whomever does the merge will squash the fixup commits.

Reply:
This disclaimer is in the tracker issue but I forgot to put it in the PR: review with care.  I haven't written Objective C since Jobs was running NeXT....